### DISCUSSION TITLE
Make it easy to record complete session as multi-track audio

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#985] [Ideas]
This is already not too difficult to do manually: You just need to change the record mode of all column tracks to "Track output" and then press REAPER's record button.

But I could add a convenience function that allows to do this at the press of a button.

**Comment by Qsonic72** [comment]
Helo Helgoboos, Thank you for you work. Playtime2 and Reaper is the most powerfull Daw i ever had! I use Playtime 2 live, and it would be really helpful to have a per-track toggle to switch between Recording into a Playtime clip slot, and Recording/printing Playtime playback onto the corresponding REAPER track.

This can already be done, also with a script, but what’s missing (and would be a big improvement for live use) is an optional per-track/per-column link between REAPER track record-arm and Playtime clip/column record-arm (Smart Record logic).

Ideally, per track/column you could choose:

clip armed only,
track armed only,
or both.

edit: To get around this, I’m now using additional monitor/recording tracks where I record the output of the Playtime tracks. This way, the clip/column record arm on the Playtime track is decoupled  from the REAPER track record arm (because that lives on the separate audio track).

Downsides of this setup:

Extra tracks, routing and trackmanagement are required. and som hassel with “Ghost” recordings appear on the Playtime track when input routing goes through Playtime — e.g. (guitar → Playtime 2 clip → REAPER audio track), especially in combination with VST FX on the Playtime track.

As far as I can see right now, being able to arm the REAPER track independently (while still playing Playtime clips on that same track) would give a lot of freedom, because it would allow clip recording and track recording to blend into each other seamlessly.
